src/shipit_static_analysis/tests/test_clang.py

```diff
@@ -173,3 +174,38 @@ def test_clang_tidy(tmpdir, mock_config, mock_stats, mock_revision):
     metrics = mock_stats.get_metrics('runtime.clang-tidy.avg')
     assert len(metrics) == 1
     assert metrics[0][1] > 0
+
+
+def test_clang_tidy_checkers():
+    '''
+    Test that all our clang-tidy checks actually exist
+    '''
+
+    # Build set of all available clang-tidy checks
+    cmd = [
+        'clang-tidy',
+        '-list-checks',
+        '-checks=*'
+    ]
+    clang_output = subprocess.check_output(cmd).decode('utf-8')
+    available_checkers = set(line.strip() for line in clang_output.split('\n'))
+
+    # Import our own clang-tidy configuration
+    from shipit_static_analysis.config import CONFIG_URL, settings
+    assert len(settings.clang_checkers) > 0, \
+        'Firefox clang-tidy configuration {} should specify > 0 clang_checkers'.format(CONFIG_URL)
+
+    # Verify that our specified clang-tidy checks actually exist
+    for checker in settings.clang_checkers:
+        name = checker['name']
```

Comment:
Nit: you can use [fnmatch](https://docs.python.org/3/library/fnmatch.html) to reduce the amount of code here.